File: fizzbuzz/FizzBuzz.cs
using System.Linq;

class FizzBuzzMain
{

    static string FizzBuzz(int n)
    {
        if (n % 15 == 0)
	    return "FizzBuzz";
	else if (n % 3 == 0)
	    return "Fizz";
	else if (n % 5 == 0)
	    return "Buzz";
	else
	    return n.ToString();
    }

    static void Main()
    {
        foreach (var i in Enumerable.Range(1, 100))
	    System.Console.WriteLine(FizzBuzz(i));
    }

}

File: fizzbuzz/FizzBuzz.scala
object Main extends App {

  def fizzbuzz(n: Int): String =
    if (n % 15 == 0)
      "FizzBuzz"
    else if (n % 3 == 0)
      "Fizz"
    else if (n % 5 == 0)
      "Buzz"
    else
      n.toString

  for (i <- 1 to 100)
    println(fizzbuzz(i))

}
